### ISSUE TITLE
Color contrast for text should be improved

### ISSUE THREAD

**Issue by rraub** [gios-asu/ASU-Web-Standards-Bootstrap#240]
Any colors we have on text should have a contrast ratio of 4.5 or greater (for AA standards). With a white background the color variable `color-boulder: #777` is just too light (4.48 ratio).  If we move to something darker like `#707070` that should get us enough contrast.

**Comment by nathan-rollins** [comment]
Should have been closed by #251. Done.
